### ISSUE TITLE
F105: Failure of Success Criterion 2.5.1 Pointer Gestures

### ISSUE THREAD

**Issue by lgnacio** [w3c/wcag#1333]
> A swipe-to-reveal control displays a set of options when swiping an item to the left, and another set of options when swiping an item to the right. One or more of these options are not available after the item is first opened with a single tap or click.

An Item can only delete with a swipe-to-reveal control  in "main screen". Can't delete an item with single tap or click, but is it successful if it  can be deleted after opened an item?

The point is, it successful if it can be deleted with single tap or click after opened an item? (not "main screen") or it should be same screen before opened an item? Clear criteria need to be presented.
